EventListener/AttachRefreshTokenOnSuccessListener.php

```diff
@@ -56,7 +60,11 @@ public function attachRefreshToken(AuthenticationSuccessEvent $event)
             $datetime->modify('+'.$this->ttl.' seconds');
 
             $refreshToken = $this->refreshTokenManager->create();
-            $refreshToken->setUsername($user->getUsername());
+
+            $accessor = new PropertyAccessor();
```

Comment:
Symfony recommandation is to use the factory (see: [PropertyComponent#Usage](https://symfony.com/doc/current/components/property_access.html#usage)) or inject the service.

But it's fine for me as it is.